azure-kusto-data/README.md

```diff
@@ -55,7 +55,7 @@ Using this method will write a token to the console, which can be used to authen
 **<!>It is not meant for production purposes<!>**
 
 ```javascript
-KustoConnectionStringBuilder.withAadUserPasswordAuthentication(`https://${clusterName}.kusto.windows.net`,'username','password');
+KustoConnectionStringBuilder.withAadDeviceAuthentication(`https://${clusterName}.kusto.windows.net`,'username','password');
```

Comment:
thanks for that. will fix it on my PR

Reply:
I have added callback to the end of invoke.